src/Plugin/os2web/DataLookup/ServiceplatformenBase.php

```diff
@@ -2,6 +2,7 @@
 
 namespace Drupal\os2web_datalookup\Plugin\os2web\DataLookup;
 
+use Drupal\Core\Extension\ExtensionPathResolver;
```

Comment:
it can be removed it we use is fully namespaced in @var tags

Reply:
Fixed in https://github.com/OS2web/os2web_datalookup/pull/9/commits/e408c62148c1305a0cb70a87e886cda822c9c882.